src/safeposix/cage.rs

```diff
@@ -128,6 +133,27 @@ impl Cage {
         *cwdbox = newwd;
     }
 
+/*     pub fn changegid(&self, newgid: i32) {
```

Comment:
this is extraneous now?

Reply:
Yes, I wrote them as a potential solution. I'll remove them as we already have a working one without changing the structure itself.